content/hardware/10.mega/shields/giga-display-shield/tutorials/13.appwizard-guide/content.md

```diff
@@ -0,0 +1,135 @@
+
+![Project with rotary](assets/appwizard-rotary.png)
+
+Now the complete project should look like this:
+
+![Complete project](assets/appwizard-complete-gui.png)
+
+Let's move on to how we can export this project to be displayed on the GIGA Display Shield.
+
+## Exporting the Project
+
+In AppWizard, go to **File -> Export & Save** in the upper left of the window. Now open the folder of the project. Create a new folder which we will put the exported files into, in this new folder create another folder named **src**. Now from the exported project folder, copy all the files contained in the **Resource** and **Source** subfolders (without copying the subfolders themselves) into the newly created **src** folder. Take all the files out of the folders before putting them in the **src** folder, if you have any folders inside of the **src** folder it will not import into the Arduino IDE later.
+
+![Folder structure](assets/appwizard-demo-folder-src.png)
+
+Now the folder name needs to be removed from the `#include` directives of the generated files. Inside the **ID_SCREEN_00_Slots.c** file change the `#include "../Generated/Resource.h"`
+and `#include "../Generated/ID_SCREEN_00.h"` to `#include "Resource.h"` and `#include "ID_SCREEN_00.h"`.
+
+![Screenshot of code](assets/appwizard-demo-folder-library-code.png)
+
+When this is done create a **library.properties** file inside the root of the newly created folder. Then put this text into the file, you can change the name to whatever you like:
+
+```arduino
+name=AppWizardDemoLib
+version=1.0.0
+author=SEGGER Microcontroller GmbH
+maintainer=SEGGER Microcontroller GmbH info@segger.com
+sentence=
+paragraph=
+category=Display
+url=
+architectures=
+includes=
+depends=
+```
+
+![Folder with library properties file](assets/appwizard-demo-folder-library.png)
+
+And lastly to create the Arduino sketch. Create an **examples** folder inside the root of the newly created folder, inside this **examples** folder you can create another folder containing the **demo.ino** file. Use the following code for the **demo.ino**:
+
+```arduino
+#include <Resource.h>
+void setup() {
+  MainTask();                 
+}
+void loop() {
+  GUI_Exec();
+}
+```
+
+![Examples folder structure](assets/appwizard-examples-folder.gif)
+
+Now all that is left is to import this whole folder as a library into the Arduino IDE. First compress the folder into a .zip format. And then in the Arduino IDE go to **Sketch -> Include library -> Add .ZIP Library**. Then select the created .zip file and add it.
+
+![Importing library in Arduino IDE](assets/appwizard-library-include.png)
+
+Now the demo sketch should be accessible under the examples tab inside the library. The library will have the name given inside the **library.properties** file.
+
+![Showing example sketch](assets/appwizard-library-example.png)
+
+Now select your board and upload the sketch. The GUI created in AppWizard should now display on the GIGA Display Shield!
+
```

Comment:
Would be nice to show a real photo of the Giga Display showing the UI.

Reply:
hello @mcmchris please use an already done image on the tutorial to use it as initial image or a quick small trick so we can move this to publish! :pray:

Reply:
Done!